statsmodels/stats/oaxaca.py

```diff
@@ -0,0 +1,254 @@
+# TODO Variance can be calculated for the three_fold
+# TODO Group Size Effects can be accounted for
+# TODO Non-Linear Oaxaca-Blinder can be used
+"""
+Author: Austin Adams
+
+This Class implements Oaxaca-Blinder Decomposition:
+
+Two-Fold/Pooled (two_fold)
+Three-Fold (three_fold)
+
+A Oaxaca-Blinder is a statistical method that is used to explain
+the differences between two mean values. The idea is to show
+from two mean values what can be explained by the data and
+what cannot by using OLS regression frameworks.
+
+"The original use by Oaxaca's was to explain the wage
+differential between two different groups of workers,
+but the method has since been applied to numerous other
+topics." (Wikipedia)
+
+The model is designed to accept two endogenous response variables
+and two exogenous explanitory variables. They are then fit using
+the specific type of decomposition that you want.
+
+The method was famously used in Card and Krueger's paper
+"School Quality and Black-White Relative Earnings: A Direct Assessment" (1992)
+
+General reference for Oaxaca-Blinder:
+
+B. Jann "The Blinder-Oaxaca decomposition for linear
+regression models," The Stata Journal, 2008.
+
+Econometrics references for regression models:
+
+E. M. Kitagawa  "Components of a Difference Between Two Rates"
+Journal of the American Statistical Association, 1955.
+
+A. S. Blinder "Wage Discrimination: Reduced Form and Structural
+Estimates," The Journal of Human Resources, 1973.
+"""
+from statsmodels.regression.linear_model import OLS
+from statsmodels.tools.tools import add_constant
+import numpy as np
+
+
+class Oaxaca(object):
```

Comment:
Is this the most common name for this procedure?  Is OaxacaBlinder or OaxacaBlinderDecomposition (or OaxacaBlinderDecomp) more obvious?

Reply:
Seems to have he same name as Stata.  Might still help to call it OaxacaDecomp or OaxacaDecomposition.

Reply:
I did OaxacaBlinder, but if you feel that it should be OaxacaDecomp, I can change it to that.